tests/transports-integrations/tests/integrations/test_openai.py

```
@@ -407,3 +411,42 @@ def test_12_error_handling_invalid_roles(self, openai_client, test_config):
         error = exc_info.value
         assert_valid_error_response(error, "tester")
         assert_error_propagation(error, "openai")
+
+    @skip_if_no_api_key("openai")
+    def test_13_streaming(self, openai_client, test_config):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_13_streaming`

Add return type annotation: `None`

(ANN201)